scripts/cybershake/auto_submit.py

```diff
@@ -35,6 +35,10 @@
     const.ProcessType.HF: const.HPC.maui,
     const.ProcessType.BB: const.HPC.maui,
     const.ProcessType.IM_calculation: const.HPC.maui,
+    const.ProcessType.IM_plot: const.HPC.mahuika,
+    const.ProcessType.rrup: const.HPC.mahuika,
```

Comment:
A cool extension would be to run things that are platform agnostic on whichever platform has a shorter queue. Or to saturate both queues.

Reply:
Sounds like it should have its own task

Reply:
Yeah. I'm just droning on.